README.md

```diff
@@ -130,43 +129,6 @@ You can find the prompt used by the LLM at the following location. When running
 
 [./src/stretch/llms/prompts/pickup_prompt.py](./src/stretch/llms/prompts/pickup_prompt.py)
 
-## Installation from Source
```

Comment:
Is this still duplicated somewhere

Reply:
Yes here: https://github.com/hello-robot/stretch_ai/pull/176/files#diff-c0f0997fc2d2605b308daa9068fb5ae3a8d7ee877a06df7f2c74f7c384ff0e14